nixos/modules/services/networking/tinc.nix

```diff
@@ -131,7 +139,8 @@ in
               Name = ${if data.name == null then "$HOST" else data.name}
               DeviceType = ${data.interfaceType}
               ${optionalString (data.ed25519PrivateKeyFile != null) "Ed25519PrivateKeyFile = ${data.ed25519PrivateKeyFile}"}
-              ${optionalString (data.listenAddress != null) "BindToAddress = ${data.listenAddress}"}
+              ${optionalString (data.listenAddress != null) "ListenAddress = ${data.listenAddress}"}
+              ${optionalString (data.bindToAddress != null) "BindToAddress = ${data.bindToAddress}"}
```

Comment:
Well yes but in that case they would hopefully go check the description of the option and realize that they need to change something.
It would be cool if nixos had a mechanism to notify you when a package/module you are using has potentially breaking changes but I don't know of anything that would do that. In the meantime, I think I'd rather break something now, than having an option that does something different than what you expect.